src/service/performance-impl.js

```diff
@@ -518,11 +533,17 @@ export class Performance {
    * Tick the largest contentful paint metrics.
    */
   tickLargestContentfulPaint_() {
+    /** @type {number|null} */ let end;
     if (this.largestContentfulPaintLoadTime_ !== null) {
       this.tickDelta('lcpl', this.largestContentfulPaintLoadTime_);
+      end = this.largestContentfulPaintLoadTime_;
     }
     if (this.largestContentfulPaintRenderTime_ !== null) {
       this.tickDelta('lcpr', this.largestContentfulPaintRenderTime_);
+      end = end || this.largestContentfulPaintRenderTime_;
+    }
+    if (end !== null) {
+      this.tickSinceVisible('lcp-v', end);
```

Comment:
Should this be two metrics, `lcpl-v` and` lcpr-v`?

Reply:
I think that the user is probably not interested in collating this data themselves, but open to be persuaded otherwise. Therefore we should rely on the more accurate measurement, and fall back to the less accurate one. The core web vitals is LCP, not LCP (load time) and LCP (render time).

Reply:
Which one is more accurate?

Reply:
Render time is preferred to load time, but load time is always available:  https://web.dev/lcp/#load-time-vs.-render-time